tools/clang/unittests/HLSLTestLib/FileCheckerTest.cpp

```diff
@@ -1091,6 +1103,59 @@ FileRunCommandResult FileRunCommandPart::RunDxilVer(dxc::DxcDllSupport& DllSuppo
   return CheckDxilVer(DllSupport, RequiredDxilMajor, RequiredDxilMinor);
 }
 
+#ifdef ENABLE_DXIL2SPV
+FileRunCommandResult
+FileRunCommandPart::RunDxil2Spv(dxc::DxcDllSupport &DllSupport,
+                                const FileRunCommandResult *Prior) {
+  // Support piping stdin from prior if needed.
+  UNREFERENCED_PARAMETER(Prior);
+  bool success = true;
+
+  std::string stdoutString;
+  std::string stderrString;
+  llvm::raw_string_ostream stdoutStream(stdoutString);
+  llvm::raw_string_ostream stderrStream(stderrString);
+
+  try {
+    // Configure filesystem.
+    llvm::sys::fs::MSFileSystem *msf;
```

Comment:
nit: can you initialize this pointer to null to be super-safe across all branches?